src/static/themes/variables.css

```diff
@@ -0,0 +1,65 @@
+:root {
```

Comment:
also export them in dynamic mode

Reply:
Already being taking care of by #2767 [see the code](https://github.com/darkreader/darkreader/blob/master/src/inject/dynamic-theme/index.ts#L101)

Reply:
those six aren't going to be enough after we get more themes than just dark and light

Reply:
Uhm, Their will be just dark and light however, the new presets can have different colors and they are being dynamically computed by the current in use preset.